### PR TITLE
Updates, fixes, and improvements based on documentation work

### LINKED ISSUE #2560: [Utilities]: Ensure grid defaults to full width if only responsive breakpoint is declared and falls below
If you only declare a responsive breakpoint in the grid, e.g. `tablet:grid-col`, it should default to 100% width once it falls below that size. Currently it defaults to the browser default which fits to the natural width of the content.

All columns should have `width: 100%` before their declared widths and media queries.

This matches the default behavior of [Bootstrap's grid](https://getbootstrap.com/docs/4.0/layout/grid/#stacked-to-horizontal).

```
.col, .col-1, .col-10, .col-11, .col-12, .col-2, .col-3, .col-4, .col-5, .col-6, .col-7, .col-8, .col-9, .col-auto, .col-lg, .col-lg-1, .col-lg-10, .col-lg-11, .col-lg-12, .col-lg-2, .col-lg-3, .col-lg-4, .col-lg-5, .col-lg-6, .col-lg-7, .col-lg-8, .col-lg-9, .col-lg-auto, .col-md, .col-md-1, .col-md-10, .col-md-11, .col-md-12, .col-md-2, .col-md-3, .col-md-4, .col-md-5, .col-md-6, .col-md-7, .col-md-8, .col-md-9, .col-md-auto, .col-sm, .col-sm-1, .col-sm-10, .col-sm-11, .col-sm-12, .col-sm-2, .col-sm-3, .col-sm-4, .col-sm-5, .col-sm-6, .col-sm-7, .col-sm-8, .col-sm-9, .col-sm-auto, .col-xl, .col-xl-1, .col-xl-10, .col-xl-11, .col-xl-12, .col-xl-2, .col-xl-3, .col-xl-4, .col-xl-5, .col-xl-6, .col-xl-7, .col-xl-8, .col-xl-9, .col-xl-auto {
  width: 100%;
}

// overridden by mq's and declared widths that come after
...
```

Do we need to declare 100% for `col-[1-12]`? bc they are the mobile sizes anyway. Or just (media*:grid-col*) ?

/cc @thisisdano

### PR DESCRIPTION
**Pull request by thisisdano**
- Add `baseline` to `flex-align` values
- Use `!important` in font assignments if the option is set
- Fix `2xl` font display bug (output this size)
- Allow `lh()` to output false values instead of erroring
- Don't use Bourbon for font stacks
- Fix `font-family` output bug (output entire stack correctly)
- Rename `u-weight` mixin to `u-font-weight`
- Add text-decoration-color as an optional prop in `u-text-decoration()`
- Allow an `!important` flag in utility mixins to output the mixin with `!important` appended
  - example: `@include u-line-height(sans, 2, !important)`
- Make responsive grid columns 100% below lowest spec'd width (fixes #2560)
- Use conventional values for `.pin` utilities — add -x and -y
  - `pin` now only positions absolute to the side or sides selected
- Rename `add-pin` utility to `pin`
- Allow `u-pin()` mixin to accept multiple values
  - example: `@include u-pin(top, x)`
- Update package-specific import guidance in project/styles.scss
  - This makes it clear that you can separate out the components, and potentially include the utilities after the custom styles and overrides
- Add basic color values to variables
  - Adds color variables (like `$red` for all the colors in the basic palette)
- Grid utilities get !important if `utilities-use-important: true`
  - Treat grid utilities like conventional utilities and use `!important` for all grid rules if the flag is set

### CODE REVIEW

**Comment by thisisdano** [comment]
OK — this is working in the docs site branch now, so I'm going to take it out of WIP.
https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/dw-componentize-docs/utilities/

File: src/stylesheets/core/_properties.scss [hunk 1]
```diff
@@ -313,6 +314,36 @@ $uswds-properties: (
       'mono-4':	   lh(mono, 4),
       'mono-5':	   lh(mono, 5),
       'mono-6':	   lh(mono, 6),
+      'cond-1':	   lh(cond, 1),
+      'cond-2':	   lh(cond, 2),
+      'cond-3':	   lh(cond, 3),
+      'cond-4':	   lh(cond, 4),
+      'cond-5':	   lh(cond, 5),
+      'cond-6':	   lh(cond, 6),
+      'heading-1': lh(heading, 1),
```
Comment (maya) [inline src/stylesheets/core/_properties.scss]: Please align values.

Comment (thisisdano) [reply src/stylesheets/core/_properties.scss]: oh huh, I see it aligned in my editor... maybe there's a hidden tab or something

**Comment by maya** [comment]
Is there a tab sneaking in? Looks larger than the space value.
screen shot 2018-06-26 at 2 09 17 pm

**Comment by thisisdano** [comment]
OK, think I got 'em this time

**Pull request merged**